src/diffusers/schedulers/scheduling_pndm_flax.py

```diff
@@ -150,16 +160,23 @@ def set_timesteps(
                 the `FlaxPNDMScheduler` state data class instance.
             num_inference_steps (`int`):
                 the number of diffusion steps used when generating samples with a pre-trained model.
-            offset (`int`):
-                optional value to shift timestep values up by. A value of 1 is used in stable diffusion for inference.
         """
+        offset = self.config.steps_offset
+
+        if "offset" in kwargs:
```

Comment:
Let's not allow this to not make the user do something "wrong" . Same comment as in DDIM